docs/sources/developers/plugins/build-a-streaming-data-source-plugin.md

```diff
@@ -95,9 +108,9 @@ Grafana uses [RxJS](https://rxjs.dev/) to continuously send data from a data sou
    };
    ```
 
-   > **Note:** In practice, you'd call `subscriber.next` as soon as you receive new data from a websocket or a message bus. The example above simulates data being received every 500 milliseconds.
+   > **Note:** In practice, you'd call `subscriber.next` as soon as you receive new data from a websocket or a message bus. In the example above, data is being received every 500 milliseconds.
 
-Here's the final `query` method.
+### Example code for final `query` method
```

Comment:
I don't know if this is linked somewhere else in this doc but we should point to https://github.com/grafana/grafana-plugin-examples/tree/main/examples/datasource-streaming-websocket for a full example of a streaming plugin

Reply:
Good idea. Will check in fix soon.

Reply:
Checked in fix, available for review.